test/test_helper.rb

```diff
@@ -26,6 +26,10 @@
 RubygemFs.mock!
 Aws.config[:stub_responses] = true
 
+Rubygem.searchkick_index.delete if Rubygem.searchkick_index.exists?
+Rubygem.searchkick_index.create
```

Comment:
@ankane would you welcome `Rubygem.searchkick_index.recreate` in Searchkick? I can provide PR.

Reply:
`reindex(import: false)` is the recommended way to recreate without data.